edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent.Test/MerticsSerializerTest.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent.Test
```

Comment:
As a general comment, the test file should live in the same directory layout as the assembly under test. I would expect this test to live in the `edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Test/serializer/MetricsSerializerTest.cs` file. We should verify all of the other test files.